1.8/usage/gpu-support.md

```diff
@@ -0,0 +1,59 @@
+---
+post_title: GPU Acceleration
+nav_title: GPU Acceleration
+feature_maturity: experimental
+menu_order: 110 
+---
+
+You can configure your DC/OS service to use the GPUs (Graphics Processing Units) of your nodes. GPUs can dramatically accelerate big data workloads. [Learn more](http://www.nvidia.com/object/what-is-gpu-computing.html).
+
+# Configure your Cluster for GPUs
+
+Follow these steps to enable GPU support on your cluster. 
+
+1. Add the `enable_gpu_isolation: true` flag to your `config.yaml` when you install DC/OS on your cluster. If you are installing DC/OS from the AWS cloud template, follow [the instructions here](/docs/1.8/administration/installing/cloud/aws/advanced/aws-custom/). Otherwise, follow [the custom installation instructions](/docs/1.8/administration/installing/custom/).
+
+1. Ensure that all machines in your cluster have [Nvidia GPU libraries](https://developer.nvidia.com/gpu-accelerated-libraries) installed on them. The machines do not need to have GPUs on them, but they will fail to start without the libraries.
```

Comment:
Oh, I also have detailed instructions on how to install them here:
https://github.com/apache/mesos/blob/master/docs/gpu-support.md#external-dependencies

If they are running on AWS though, they can simply use one of these AMIs instead of the default DC/OS one:
```
us-west-2: ami-9b5d97fb
us-east-1: ami-e10e50f6
ap-southeast-2: ami-37b28f54
```